### ISSUE TITLE
[BUG] CreateDocument exception

### ISSUE THREAD

**Issue by andrewyunt** [affinda/affinda-dotnet#4]
**Describe the bug**
SampleProgram throws an exception on latest affinda-dotnet version (updated to use CreateDocument instead of CreateResume). File is successfully uploaded to Affinda despite the exception, though this prevents us from getting data back from the parse.

**To Reproduce**
1. Update Affinda to latest version (5.1.0)
2. In SampleProgram/Program.cs:
  -  Set credential and documentPath variables
  - Change CreateResume to CreateDocument
3. Run sample program

**Expected behavior**
CreateDocument should successfully upload the file to Affinda AND return an instance of Affinda.API.Models.Document

**Stacktrace / Screenshts**
```
Unhandled exception. System.InvalidOperationException: The requested operation requires an element of type 'Object', but the target element has type 'String'.
   at System.Text.Json.JsonElement.EnumerateObject()
   at Affinda.API.Models.ComponentsEyyf0ZSchemasResumedataAdditionalproperties.DeserializeComponentsEyyf0ZSchemasResumedataAdditionalproperties(JsonElement element) in F:\Home\Projects\Resume-Raise\test\affinda-dotnet-master\AffindaAPI\AffindaAPI\Models\ComponentsEyyf0ZSchemasResumedataAdditionalproperties.Serialization.cs:line 23
   at Affinda.API.Models.ResumeData.DeserializeResumeData(JsonElement element) in F:\Home\Projects\Resume-Raise\test\affinda-dotnet-master\AffindaAPI\AffindaAPI\Models\ResumeData.Serialization.cs:line 509
   at Affinda.API.Models.Resume.DeserializeResume(JsonElement element) in F:\Home\Projects\Resume-Raise\test\affinda-dotnet-master\AffindaAPI\AffindaAPI\Models\Resume.Serialization.cs:line 57
   at Affinda.API.Models.Document.DeserializeDocument(JsonElement element) in F:\Home\Projects\Resume-Raise\test\affinda-dotnet-master\AffindaAPI\AffindaAPI\Models\Document.Serialization.cs:line 38
   at Affinda.API.AffindaAPIRestClient.CreateDocument(Stream file, String url, String collection, String workspace, String wait, String identifier, String fileName, String expiryTime, String language, String rejectDuplicates, CancellationToken cancellationToken) in F:\Home\Projects\Resume-Raise\test\affinda-dotnet-master\AffindaAPI\AffindaAPI\AffindaAPIRestClient.cs:line 1334
   at Affinda.API.AffindaAPIClient.CreateDocument(Stream file, String url, String collection, String workspace, String wait, String identifier, String fileName, String expiryTime, String language, String rejectDuplicates, CancellationToken cancellationToken) in F:\Home\Projects\Resume-Raise\test\affinda-dotnet-master\AffindaAPI\AffindaAPI\AffindaAPIClient.cs:line 714
   at AffindaAPISample.Program.Main(String[] args) in F:\Home\Projects\Resume-Raise\test\affinda-dotnet-master\SampleProgram\Program.cs:line 17
```

**Environment**
 - Client version: 5.1.0
 - .NET version 5.0.0

**Additional context**
Looking to implement this as a critical part of our application, and had issues with both affinda-dotnet 4.x.x versions and 5.x.x versions. The original CreateResume method and new CreateDocument method both did not work from our testing. Seems that this has been an issue for a while.

**Comment by andrewyunt** [comment]
@cfculhane

**Comment by cfculhane** [comment]
Thanks @andrewyunt - looking into now

Edit - I think this was fixed in 5.2.0 released yesterday as I was having the same issue but should be resolved now, can you confirm?

**Comment by andrewyunt** [comment]
Yes this is fixed, thank you!
